Started by timer
Running in Durability level: MAX_SURVIVABILITY
[8mha:////4DOlrZhma2TKYqx8mtRBwGXVE8Xt4z3OJyo/zi+ICDdzAAAAoh+LCAAAAAAAAP9tjTEOwjAQBM8BClpKHuFItIiK1krDC0x8GCfWnbEdkooX8TX+gCESFVvtrLSa5wtWKcKBo5UdUu8otU4GP9jS5Mixv3geZcdn2TIl9igbHBs2eJyx4YwwR1SwULBGaj0nRzbDRnX6rmuvydanHMu2V1A5c4MHCFXMWcf8hSnC9jqYxPTz/BXAFEIGsfuclm8zQVqFvQAAAA==[0m[Pipeline] Start of Pipeline
[8mha:////4B4K7nOhvaaLmbeE3w0NEhE1GAALwLDjakw9plXgkY6XAAAApR+LCAAAAAAAAP9tjTEOwjAUQ3+KOrAycohUghExsUZZOEFIQkgb/d8mKe3EibgadyBQiQlLlmxL1nu+oE4RjhQdby12HpP2vA+jK4lPFLtroIm3dOGaMFGwXNpJkrGnpUrKFhaxClYC1hZ1oOTRZdiIVt1VExS65pxj2Q4CKm8GeAAThZxVzN8yR9jeRpMIf5y/AJj7DGxXvP/86jduZBmjwAAAAA==[0m[Pipeline] node
Still waiting to schedule task
Waiting for next available executor on â€˜[8mha:////4C8PgMHPegRpCg8+HpIiIHK02uxvuQktJylkqRHkJagNAAAAox+LCAAAAAAAAP9b85aBtbiIQTGjNKU4P08vOT+vOD8nVc83PyU1x6OyILUoJzMv2y+/JJUBAhiZGBgqihhk0NSjKDWzXb3RdlLBUSYGJk8GtpzUvPSSDB8G5tKinBIGIZ+sxLJE/ZzEvHT94JKizLx0a6BxUmjGOUNodHsLgAzeEgYJ/eT83ILSktQiICOvpCg/Jye1SNfSUh8ABjlyb80AAAA=[0mcontroller-99â€™
Running on [8mha:////4C8PgMHPegRpCg8+HpIiIHK02uxvuQktJylkqRHkJagNAAAAox+LCAAAAAAAAP9b85aBtbiIQTGjNKU4P08vOT+vOD8nVc83PyU1x6OyILUoJzMv2y+/JJUBAhiZGBgqihhk0NSjKDWzXb3RdlLBUSYGJk8GtpzUvPSSDB8G5tKinBIGIZ+sxLJE/ZzEvHT94JKizLx0a6BxUmjGOUNodHsLgAzeEgYJ/eT83ILSktQiICOvpCg/Jye1SNfSUh8ABjlyb80AAAA=[0mcontroller-99 in /root/.jenkins/workspace/test-test-nic-sriov-5times
[8mha:////4NUxBMzN6CtFAbQ4pdo7pH8ZuwVHigLf0SdP3FcjQTwyAAAApR+LCAAAAAAAAP9tjTEOwjAUQ3+KOrAycoh0gA0xsUZZOEFIQkgb/d8mKe3EibgadyBQiQlLlmxL1nu+oE4RjhQdby12HpP2vA+jK4lPFLtroIm3dOGaMFGwXNpJkrGnpUrKFhaxClYC1hZ1oOTRZdiIVt1VExS65pxj2Q4CKm8GeAAThZxVzN8yR9jeRpMIf5y/AJj7DGxXvP/86jfoP95RwAAAAA==[0m[Pipeline] {
[8mha:////4OCB1+Up79PmV3LnMeomKr9r+JFCKkFjAfTpr2UorbLOAAAApR+LCAAAAAAAAP9tjTEOwjAUQ3+KOrAycoh0gQkxsUZZOEFIQkgb/d8mKe3EibgadyBQiQlLlmxL1nu+oE4RjhQdby12HpP2vA+jK4lPFLtroIm3dOGaMFGwXNpJkrGnpUrKFhaxClYC1hZ1oOTRZdiIVt1VExS65pxj2Q4CKm8GeAAThZxVzN8yR9jeRpMIf5y/AJj7DGxXvP/86jc09154wAAAAA==[0m[Pipeline] stage
[8mha:////4N6KQN35K8CT/Pcr9a+R4oOEu4uEh3phvpB1t67zd2hmAAAApR+LCAAAAAAAAP9tjTEOwjAUQ3+KOrAycoh0ggUxsUZZOEFIQkgb/d8mKe3EibgadyBQiQlLlmxL1nu+oE4RjhQdby12HpP2vA+jK4lPFLtroIm3dOGaMFGwXNpJkrGnpUrKFhaxClYC1hZ1oOTRZdiIVt1VExS65pxj2Q4CKm8GeAAThZxVzN8yR9jeRpMIf5y/AJj7DGxXvP/86jek7ggRwAAAAA==[0m[Pipeline] { (rm project dir)
[8mha:////4EWjgIQS1R18e39T6LdOu3ZpSMSXMburmKwxJHd8e163AAAApR+LCAAAAAAAAP9tjTEOwjAUQ3+KOrAycoh0gwExsUZZOEFIQkgb/d8mKe3EibgadyBQiQlLlmxL1nu+oE4RjhQdby12HpP2vA+jK4lPFLtroIm3dOGaMFGwXNpJkrGnpUrKFhaxClYC1hZ1oOTRZdiIVt1VExS65pxj2Q4CKm8GeAAThZxVzN8yR9jeRpMIf5y/AJj7DGxXvP/86jcChmMxwAAAAA==[0m[Pipeline] script
[8mha:////4NOJ/YVWDlnnBpjDdLKlLZkF5LrR387vblsx7fWhZkPlAAAApR+LCAAAAAAAAP9tjTEOwjAUQ3+KOrAycoh0BAkxsUZZOEFIQkgb/d8mKe3EibgadyBQiQlLlmxL1nu+oE4RjhQdby12HpP2vA+jK4lPFLtroIm3dOGaMFGwXNpJkrGnpUrKFhaxClYC1hZ1oOTRZdiIVt1VExS65pxj2Q4CKm8GeAAThZxVzN8yR9jeRpMIf5y/AJj7DGxXvP/86jfpX/cvwAAAAA==[0m[Pipeline] {
[8mha:////4MAm0moKTc8Eo2PcB0LgkHNHI7Xtf5Poej2Mf/l8xh/lAAAAoh+LCAAAAAAAAP9tjTEOAiEURD9rLGwtPQTbaoyVLaHxBMgiwpL/WWDdrTyRV/MOEjexcpJJ5k3zXm9Y5wQnSpZ7g73DrB2PYbR18YlSfws0cU9XrgkzBcOlmSR15rygpGJgCWtgJWBjUAfKDm2BrfDqodqg0LaXkup3FNC4boAnMFHNRaXyhTnB7j52mfDn+SuAORZg+9pD/AAeoCAqvAAAAA==[0m[Pipeline] sh
+ ls /root/.jenkins/workspace/test-test-nic-sriov-5times/npxtester
1.0.0.0001.version
conf
installation
libs
log_analyzer
README.md
testconf
testLibs
Tests
[8mha:////4F3BY6hez6Yn9lSF4dYULOlz4RwK0Fdsp1OYWtj0vxDtAAAAox+LCAAAAAAAAP9tjTEOwjAQBDdBFLSUPMKBDglR0VppeIFJjHFi3QX7QlLxIr7GH4iIRMVWO9PM641lijhydKqx1HpKlVdd6N301MCxvQYeVMMXVTElDlaVdii5tqcZSxaLeVmOhcbKUhU4eXKCtW7MwxTBkCvOEid30Mh9fccTmZ7KYqJ8YYzY3Po6Mf06fwMYO0G2F+S7bfcBITL9lL0AAAA=[0m[Pipeline] echo
dir is exists and rm dir
[8mha:////4KL1cTFlC/t8/0LRf71hJwxXIUFLIMviWtcIHXR2W8epAAAAox+LCAAAAAAAAP9tjbEOgjAURS8YB1dHP6KEzcQ4uTYsfkGFWgvNe9g+hMkv8tf8B4kkTt7pnrOc1xvrFHHk6FRrqfOUaq/6MLj5qZFjdw08qpYvqmZKHKyq7FhxY08LViwWy7IcK42NpTpw8uQEW92ahymCIVecJc7uoJH75o4nMj2XxUT5whSxuw1NYvp1/gYw9YJsL8jLsv8AYAPmjb0AAAA=[0m[Pipeline] sh
+ rm -rf /root/.jenkins/workspace/test-test-nic-sriov-5times/npxtester
[8mha:////4GzJynz9IX3CLl34e/1JpYOeLz7FDsqMtIihCtFGqNpZAAAAox+LCAAAAAAAAP9tjTEOwjAQBDdBFLSUPMIRVEiIitZKwwtMYowT6y7YF5KKF/E1/kBEJCq22plmXm8sU8SRo1ONpdZTqrzqQu+mpwaO7TXwoBq+qIopcbCqtEPJtT3NWLJYzMtyLDRWlqrAyZMTrHVjHqYIhlxxlji5g0bu6zueyPRUFhPlC2PE5tbXienX+RvA2HWCfLsTZPsPzjQzob0AAAA=[0m[Pipeline] }
[8mha:////4MvApYhNLg1xEFG2iXyHCog7Z9h4MbdAsNSDoK4wJnDgAAAApB+LCAAAAAAAAP9tjTEOwjAQBDdBFLSUPMIRoqBAVLRWGl5gEmOcWHfBvpBUvIiv8QciIlGx1c4083pjmSKOHJ1qLLWeUuVVF3o3PTVwbK+BB9XwRVVMiYNVpR1Kru1pxpLFYl6WY6GxslQFTp6cYK0b8zBFMOSKs8TJHTRyX9/xRKanspgoXxgjNre+Tky/zt8Axq4T5NudINt/AO8A7Ay9AAAA[0m[Pipeline] // script
[8mha:////4P1g47DHQeYIMdAQPrqVFJW5XTEYSJkMqP0k+p1upCWxAAAApB+LCAAAAAAAAP9tjTEOwjAQBDdBFLSUPMIREqJBVLRWGl5gEmOcWHfBvpBUvIiv8QciIlGx1c4083pjmSKOHJ1qLLWeUuVVF3o3PTVwbK+BB9XwRVVMiYNVpR1Kru1pxpLFYl6WY6GxslQFTp6cYK0b8zBFMOSKs8TJHTRyX9/xRKanspgoXxgjNre+Tky/zt8Axq4T5NudINt/AGnsy8m9AAAA[0m[Pipeline] }
[8mha:////4Lc6WrLYEiiAC7rTWY8q4RQPGu1FD+hrgClvEqpzcg2sAAAAox+LCAAAAAAAAP9tjTESgjAQRT84FraWHiKMBZVjZZuh8QQRYgxkdjFZhMoTeTXvICMzVv7qv9e81xvrFHHk6FRrqfOUaq/6MLj5qZFjdw08qpYvqmZKHKyq7FhxY08LViwWy7IcK42NpTpw8uQEW92ahymCIVecJc7uoJH75o4nMj2XxUT5whSxuw1NYvp1/gYw9b0g35eCrPwAY5Sibb0AAAA=[0m[Pipeline] // stage
[8mha:////4LPzYIRsJlP9vBl87WR2FbvV5590n9jZkqHCRBijPEs5AAAApx+LCAAAAAAAAP9tjTEOwjAQBM9BKWgpeYQjJESDqGgtN7zAxMY4se6CfSGpeBFf4w8EIlGx1c5Kq3m+oMwJDpS8bBy2AXMdZBd7PzU5UGovkQbZ0FnWhJmik9oNmqw7zqiJHcwRBSwULB3WkXJAz7BSjbmbKhr01YnTtO0VFMHe4AFCTWY2ib8wJlhfe5sJf56/Ahg7BrFlKDa7z7F8A6sHc4fBAAAA[0m[Pipeline] stage
[8mha:////4P0yjWfQiEwQxZO7i9obwDXrYwQclNR3i//1CTGbFwBFAAAApx+LCAAAAAAAAP9tjTEOwjAUQ3+LOrAycohULDAgpq5RFk4QmhDSRv+Xn5R24kRcjTvQUokJD5ZtyXqvNxSR4UTsRGOx9RhrL7rQuymJgbi9BhpEQxdRE0YKVig7KDK2WqqiZGFRlsNKwtpiHSh6dAk2stEPXQaNrjwnnrajhNybOzwhkxM5aU7fMjJsb72JhD/OXwCMXYJ8t5/tMD+LD1+IXOLCAAAA[0m[Pipeline] { (git clone)
[8mha:////4I3tKNYAz7At397ZkNOs6SFwEn2bzn3ZBhsoegW1dhRwAAAApB+LCAAAAAAAAP9tjTEOwjAQBC9BFLSUPOIiKpAQVVrLDS8wsTFOrDtjOyQVL+Jr/IFAJCq2WO1sM88XLFOEI0eLraHOUWocBt/baeHAsbt4HrDlMzZMib1BaQbJ2tQzSs4G5hQlLASsDDWekyObYS1adVeVV2SrU47TdxBQOn2DBxRiMmcV8xfGCJtrrxPTz/NXAGPIUG53n9qHNxXfVKy+AAAA[0m[Pipeline] sh
+ git clone -b develop https://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@github.com/intel-innersource/applications.validation.network-platform.npxtester.git npxtester
Cloning into 'npxtester'...
[8mha:////4CRWXV91kUhqh2bTVXI+2n5U32cQ89nJ7yub5blKQ29bAAAApB+LCAAAAAAAAP9tjTEOwjAQBC9BFLSUPOIiKoQQVVrLDS8wsTFOrDtjOyQVL+Jr/IFAJCq2WO1sM88XLFOEI0eLraHOUWocBt/baeHAsbt4HrDlMzZMib1BaQbJ2tQzSs4G5hQlLASsDDWekyObYS1adVeVV2SrU47TdxBQOn2DBxRiMmcV8xfGCJtrrxPTz/NXAGMIGcrt/lO7N9Jjga++AAAA[0m[Pipeline] }
[8mha:////4I4ZVrOFMvh2sVfxnfT9s6+paGHoH9OzkxEwdR4h+nRDAAAApR+LCAAAAAAAAP9tjTEOwjAQBC9BFLSUPOICFDQoFa3lhheYxBgn1l2wHZyKF/E1/oBFJCq2WO1sM683LIOHmr3BTlNvKTQWBzeavDCx76+OE3Z8wYYpsNModZLc6tOMkqOGOUUJCwErTY3jYMlEWItOPVTlFJnqHH3+jgJK297hCYXI5qh8/MLkYXMb28D08/wVwDQMEcr9Ntfu8AERGGHOvgAAAA==[0m[Pipeline] // stage
[8mha:////4J4BpfnRrLE6odVrNZGgZ11wIRy0HYHBalvcHAcD12j/AAAAph+LCAAAAAAAAP9tjTEOwjAQBM9BKWgpeYQjEB2iorXc8AITG+PEugv2haTiRXyNPxCIRMVWOyut5vmCMic4UPKycdgGzHWQXez91ORAqb1EGmRDZ1kTZopOajdosu44oyZ2MEcUsFCwdFhHygE9w0o15m6qaNBXJ07TtldQBHuDBwg1mdkk/sKYYH3tbSb8ef4KYOwYxI6h2G4+x/INji2eq8EAAAA=[0m[Pipeline] stage
[8mha:////4AgPhGmio4pMNzq5gItmuDxswb3P5dwyIofUIBdMXBTmAAAAph+LCAAAAAAAAP9tjTEOwjAUQ3+DOrAycohUdEVMrFEXThCaEJJG/7dJSjpxIq7GHWipxIQHy7ZkvdcbyhjgRMFwp7GzGFvLez+aOfFMobt5ytzRlbeEkbzmjc4NKX1ea0NJw6qCwUbAVmPrKVo0CXbCyYesvERTXVKYt6MAZtUATyjETE4ypG+ZAuzvo4qEP85fAEx9AlYfFquXZ/kBAsTc3cIAAAA=[0m[Pipeline] { (pre-exec-set-args)
[8mha:////4G9dxluK5ut2lxdfo2JVi6OIov1ZH/881VUFzuPzanonAAAAox+LCAAAAAAAAP9tjTESwiAURH/iWNhaeggyE0vHypah8QSYIEKY/wkQSeWJvJp3EM2MlVvs7NvmPV+wjgGOFDSzCgeDsTPMu0mXxTKF4eooM0sX1hFGcooJlQX16rSgoKRgSVXDisNGYecoGtQJttzKu2ycRN2cUyjfgUNt+hEeUPFiTjKkL8wBdrepj4Q/z18BzD5B3baf2vs3w5BRkb4AAAA=[0m[Pipeline] sh
+ python /root/.jenkins/workspace/test-test-nic-sriov-5times/npxtester/conf/change_args.py '{"drive_info":{"drive_version":"1.6.4"},"dpdk_dir":"/opt/APP/utility/DPDK_install/dpdk-stable-20.11.1/","vm_image_conf":{"image1":"rhel.img","image2":"rhel2.img","image_dir":"/home/image"},"sut_conf":{"host":"10.112.97.62","pf1":"ens2f0","os":"RHEL"},"pkg_generator":{"host":"10.112.97.60","pf1":"ens2f1"}}'
[8mha:////4FhUQONKPXFSeOjFO92QKbtxv93cUzSuhB7hx3xoWX6KAAAApB+LCAAAAAAAAP9tjbEOwiAURV9rHFwd/QiaNG7GyZWw+AXYIkLJexSodPKL/DX/QbSJk3e4uecu5/mCdQxwpKCZVTgYjJ1h3k26LJYpDFdHmVm6sI4wklNMqCyoV6cFBSUFS6oaVhw2CjtH0aBOsOVW3mXjJOrmnEL5Dhxq04/wgIoXc5IhfWEOsLtNfST8ef4KYPY+Qd3uP9W+Ad3NsHm+AAAA[0m[Pipeline] }
[8mha:////4EpM1Tihe33kOTBucrClpdwMCq1FF6wGwTU/AEOZmYaNAAAApB+LCAAAAAAAAP9tjTEOwjAQBC9BFLSUPOIikKgQVVrLDS8wsTFOrDtjOyQVL+Jr/IFAJCq2WO1sM88XLFOEI0eLraHOUWocBt/baeHAsbt4HrDlMzZMib1BaQbJ2tQzSs4G5hQlLASsDDWekyObYS1adVeVV2SrU47TdxBQOn2DBxRiMmcV8xfGCJtrrxPTz/NXAGMIGcrd/lPbN8JP5Su+AAAA[0m[Pipeline] // stage
[8mha:////4JJk4NQMkVX21N+ktknOqKLQw+FD2VU/fvD8dhBPuambAAAApx+LCAAAAAAAAP9tjTEOwjAQBM9BKWgpeYQjIUSDqGgtN7zAxMY4se6CfSGpeBFf4w8EIlGx1c5Kq3m+oMwJDpS8bBy2AXMdZBd7PzU5UGovkQbZ0FnWhJmik9oNmqw7zqiJHcwRBSwULB3WkXJAz7BSjbmbKhr01YnTtO0VFMHe4AFCTWY2ib8wJlhfe5sJf56/Ahg7BrFlKDa7z7F8AzYdm7bBAAAA[0m[Pipeline] stage
[8mha:////4GToWcz1lEtttcc1F67+obVKf97ChyGWoNBgdNb8ln6iAAAAqB+LCAAAAAAAAP9tjTEOwjAUQ3+LOrAycohUYoABMXWNsnCC0ISQNvq//KS0EyfiatyBlkpMeLBsS9Z7vaGIDCdiJxqLrcdYe9GF3k1JDMTtNdAgGrqImjBSsELZQZGx1VIVJQuLshxWEtYW60DRo0uwkY1+6DJodOU58bQdJeTe3OEJmZzISXP6lpFhe+tNJPxx/gJg7BLku/1sh/lZfAAyQCqkwgAAAA==[0m[Pipeline] { (run nic case)
[8mha:////4M/BBJOrvLliK1a64sS7A0hf4H4seiclg8D1IX9OqKAVAAAApR+LCAAAAAAAAP9tjTEOwjAQBC9BFLSUPOIi0YCEqNJabniBiY1xYt0Z2yGpeBFf4w8EIlGxxWpnm3m+YJkiHDlabA11jlLjMPjeTgsHjt3F84Atn7FhSuwNSjNI1qaeUXI2MKcoYSFgZajxnBzZDGvRqruqvCJbnXKcvoOA0ukbPKAQkzmrmL8wRthce52Yfp6/AhhDhnK7+9Q+vAGvZp0gvgAAAA==[0m[Pipeline] sh
+ pytest /root/.jenkins/workspace/test-test-nic-sriov-5times/npxtester/Tests/testNics -s -m nicSriov5times --capture=tee-sys --html=/root/testconfig/testlogs/HtmlReport/nic_sriov5times_case_report.html --self-contained-html
============================= test session starts ==============================
platform linux -- Python 3.6.8, pytest-6.2.4, py-1.11.0, pluggy-0.13.1
rootdir: /root/.jenkins/workspace/test-test-nic-sriov-5times/npxtester/Tests/testNics, configfile: pytest.ini
plugins: metadata-1.11.0, html-2.1.1
collected 20 items / 19 deselected / 1 selected

npxtester/Tests/testNics/test_network_sriov_stress.py 2022-04-26 02:51:02 - NicTestFixture.py - [line:27] - root - INFO - case_logpath: /root/testconfig/testlogs/2022-04-26/test_PI_Networking_SRIOV_StressDisableEnableFor5TimesWhenVFIsAssigned_VM_Guest[rhel.img-rhel2.img]_02_51_02/test_PI_Networking_SRIOV_StressDisableEnableFor5TimesWhenVFIsAssigned_VM_Guest[rhel.img-rhel2.img].log, case_logdir: /root/testconfig/testlogs/2022-04-26/test_PI_Networking_SRIOV_StressDisableEnableFor5TimesWhenVFIsAssigned_VM_Guest[rhel.img-rhel2.img]_02_51_02
2022-04-26 02:51:02 - NicTestFixture.py - [line:31] - root - INFO - ============================ Starting to run test_PI_Networking_SRIOV_StressDisableEnableFor5TimesWhenVFIsAssigned_VM_Guest[rhel.img-rhel2.img] case ============================
2022-04-26 02:51:02 - test_network_sriov_stress.py - [line:32] - root - INFO - SSH SUT and setup configurations in the setup function.
2022-04-26 02:51:02 - test_network_sriov_stress.py - [line:50] - root - INFO - ssh to the sut1
2022-04-26 02:51:02 - transport.py - [line:1819] - paramiko.transport - INFO - Connected (version 2.0, client OpenSSH_8.0)
2022-04-26 02:51:02 - transport.py - [line:1819] - paramiko.transport - INFO - Authentication (password) successful!
2022-04-26 02:51:02 - sshInstance.py - [line:28] - root - INFO - SSH on SUT: 10.112.97.62
2022-04-26 02:51:02 - transport.py - [line:1819] - paramiko.transport - INFO - Connected (version 2.0, client OpenSSH_8.0)
2022-04-26 02:51:02 - transport.py - [line:1819] - paramiko.transport - INFO - Authentication (password) successful!
2022-04-26 02:51:02 - sshInstance.py - [line:28] - root - INFO - SSH on SUT: 10.112.97.60
2022-04-26 02:51:02 - test_network_sriov_stress.py - [line:64] - root - INFO - clean up iperf and taskset process...
2022-04-26 02:51:05 - sshInstance.py - [line:50] - root - INFO - pkill -9 iperf
2022-04-26 02:51:05 - sshInstance.py - [line:54] - root - INFO - 
2022-04-26 02:51:13 - sshInstance.py - [line:50] - root - INFO - pkill -9 taskset
2022-04-26 02:51:13 - sshInstance.py - [line:54] - root - INFO - 
2022-04-26 02:51:21 - sshInstance.py - [line:50] - root - INFO - pkill -9 iperf
2022-04-26 02:51:21 - sshInstance.py - [line:54] - root - INFO - 
2022-04-26 02:51:30 - sshInstance.py - [line:50] - root - INFO - pkill -9 taskset
2022-04-26 02:51:30 - sshInstance.py - [line:54] - root - INFO - 
2022-04-26 02:51:35 - sftp.py - [line:158] - paramiko.transport.sftp - INFO - [chan 3] Opened sftp connection (server version 3)
2022-04-26 02:51:35 - sshInstance.py - [line:199] - root - INFO - start to rmmod irdma....
2022-04-26 02:51:38 - sshInstance.py - [line:50] - root - INFO - rmmod irdma
2022-04-26 02:51:38 - sshInstance.py - [line:54] - root - INFO - rmmod: ERROR: Module irdma is not currently loaded

2022-04-26 02:51:43 - sshInstance.py - [line:202] - root - INFO - finished rmmod irdma....
2022-04-26 02:51:43 - sshInstance.py - [line:203] - root - INFO - start to rmmod ice....
2022-04-26 02:51:46 - sshInstance.py - [line:50] - root - INFO - rmmod ice
2022-04-26 02:51:50 - sshInstance.py - [line:54] - root - INFO - 
2022-04-26 02:51:55 - sshInstance.py - [line:206] - root - INFO - finished rmmod ice....
2022-04-26 02:51:55 - sshInstance.py - [line:208] - root - INFO - start to modprobe ice....
2022-04-26 02:51:58 - sshInstance.py - [line:50] - root - INFO - modprobe ice
2022-04-26 02:52:03 - sshInstance.py - [line:54] - root - INFO - 
2022-04-26 02:52:08 - sshInstance.py - [line:210] - root - INFO - finished modprobe ice....
2022-04-26 02:52:11 - sshInstance.py - [line:50] - root - INFO - ethtool -i ens2f0
2022-04-26 02:52:11 - sshInstance.py - [line:54] - root - INFO - driver: ice
version: 1.6.4
firmware-version: 2.50 0x80006a16 1.2852.0
expansion-rom-version: 
bus-info: 0000:4b:00.0
supports-statistics: yes
supports-test: yes
supports-eeprom-access: yes
supports-register-dump: yes
supports-priv-flags: yes

2022-04-26 02:52:16 - test_network_sriov_stress.py - [line:72] - root - INFO - try to get ipv6 of pf for redhat...
2022-04-26 02:52:16 - sshInstance.py - [line:187] - root - INFO - starting to delete network of ens2f0:
2022-04-26 02:52:22 - sshInstance.py - [line:99] - root - INFO - Activate the web console with: systemctl enable --now cockpit.socket
This system is not registered to Red Hat Insights. See https://cloud.redhat.com/
To register this system, run: insights-client --register
Last login: Tue Apr 26 02:52:05 2022 from 10.239.182.99
[root@WW25ICX2 ~]# nmcli connection delete ens2f0
Connection 'ens2f0' (54297e57-5d35-497f-983f-8ec5be76931a) successfully deleted.
[root@WW25ICX2 ~]# 
2022-04-26 02:52:25 - sshInstance.py - [line:189] - root - INFO - restart networkManager to make setting valid
2022-04-26 02:52:31 - sshInstance.py - [line:99] - root - INFO - systemctl restart NetworkManager
[root@WW25ICX2 ~]# 
2022-04-26 02:52:34 - sshInstance.py - [line:192] - root - INFO - starting to add network of ens2f0:
2022-04-26 02:52:40 - sshInstance.py - [line:99] - root - INFO - nmcli connection add type ethernet con-name ens2f0 ifname ens22f0
Connection 'ens2f0' (c5db0ab6-a125-4b1a-938b-99c2071a32e3) successfully added.
[root@WW25ICX2 ~]# 
2022-04-26 02:52:43 - sshInstance.py - [line:195] - root - INFO - restart networkManager to make setting valid
2022-04-26 02:52:49 - sshInstance.py - [line:99] - root - INFO - systemctl restart NetworkManager
[root@WW25ICX2 ~]# 
2022-04-26 02:52:52 - test_network_sriov_stress.py - [line:78] - root - INFO - try to get ipv6 for ens2f1
2022-04-26 02:52:52 - test_network_sriov_stress.py - [line:80] - root - INFO - getting ipv6 in redhat.
2022-04-26 02:52:52 - sshInstance.py - [line:187] - root - INFO - starting to delete network of ens2f1:
2022-04-26 02:52:58 - sshInstance.py - [line:99] - root - INFO - Activate the web console with: systemctl enable --now cockpit.socket
This system is not registered to Red Hat Insights. See https://cloud.redhat.com/
To register this system, run: insights-client --register
Last login: Tue Apr 26 02:44:09 2022 from 10.239.182.99
[root@localhost ~]# nmcli connection delete ens2f1
Connection 'ens2f1' (c9195f30-87be-4e58-8ae5-ed1c588eed53) successfully deleted.
[root@localhost ~]# 
2022-04-26 02:53:01 - sshInstance.py - [line:189] - root - INFO - restart networkManager to make setting valid
2022-04-26 02:53:07 - sshInstance.py - [line:99] - root - INFO - systemctl restart NetworkManager
[root@localhost ~]# 
2022-04-26 02:53:10 - sshInstance.py - [line:192] - root - INFO - starting to add network of ens2f1:
2022-04-26 02:53:16 - sshInstance.py - [line:99] - root - INFO - nmcli connection add type ethernet con-name ens2f1 ifname enss2f1
Connection 'ens2f1' (49e80887-0bb9-420c-8e17-9a7504951e09) successfully added.
[root@localhost ~]# 
2022-04-26 02:53:19 - sshInstance.py - [line:195] - root - INFO - restart networkManager to make setting valid
2022-04-26 02:53:25 - sshInstance.py - [line:99] - root - INFO - systemctl restart NetworkManager
[root@localhost ~]# 
2022-04-26 02:53:28 - test_network_sriov_stress.py - [line:88] - root - INFO - trying to get ipv6 addr for pf1
2022-04-26 02:53:31 - sshInstance.py - [line:50] - root - INFO - ifconfig ens2f0
2022-04-26 02:53:31 - sshInstance.py - [line:54] - root - INFO - ens2f0: flags=4163<UP,BROADCAST,RUNNING,MULTICAST>  mtu 1500
        inet6 fe80::cfa4:396e:1144:775d  prefixlen 64  scopeid 0x20<link>
        ether b4:96:91:9c:f7:a4  txqueuelen 1000  (Ethernet)
        RX packets 0  bytes 0 (0.0 B)
        RX errors 0  dropped 0  overruns 0  frame 0
        TX packets 46  bytes 7911 (7.7 KiB)
        TX errors 0  dropped 0 overruns 0  carrier 0  collisions 0


2022-04-26 02:53:39 - sshInstance.py - [line:50] - root - INFO - ifconfig ens2f1
2022-04-26 02:53:39 - sshInstance.py - [line:54] - root - INFO - ens2f1: flags=4163<UP,BROADCAST,RUNNING,MULTICAST>  mtu 1500
        inet 192.168.10.11  netmask 255.255.255.0  broadcast 192.168.10.255
        inet6 fe80::b696:91ff:fea5:9b45  prefixlen 64  scopeid 0x20<link>
        ether b4:96:91:a5:9b:45  txqueuelen 1000  (Ethernet)
        RX packets 494489227  bytes 748651576084 (697.2 GiB)
        RX errors 0  dropped 0  overruns 0  frame 0
        TX packets 19779733  bytes 1378371080 (1.2 GiB)
        TX errors 0  dropped 0 overruns 0  carrier 0  collisions 0


2022-04-26 02:53:44 - test_network_sriov_stress.py - [line:604] - root - INFO - 
Stress Disable and Enable for 1 time when VF is Assigned VM guest.

2022-04-26 02:53:47 - sshInstance.py - [line:50] - root - INFO - /opt/APP/utility/DPDK_install/dpdk-stable-20.11.1/usertools/dpdk-devbind.py -s | grep ens2f0 | awk '{print $1;}'
2022-04-26 02:53:48 - sshInstance.py - [line:54] - root - INFO - 0000:4b:00.0

2022-04-26 02:53:53 - test_network_sriov_stress.py - [line:139] - root - INFO - pf_id was found: 0000:4b:00.0

2022-04-26 02:53:53 - test_network_sriov_stress.py - [line:612] - root - INFO - format_pf_id is 0000\:4b\:00.0

2022-04-26 02:53:53 - test_network_sriov_stress.py - [line:615] - root - INFO - pf_id_key: 4b
2022-04-26 02:53:53 - test_network_sriov_stress.py - [line:619] - root - INFO - start to assign ip for ens2f0 of SUT1: 192.168.10.10
2022-04-26 02:53:59 - sshInstance.py - [line:99] - root - INFO - ifconfig ens2f0 192.168.10.10
[root@WW25ICX2 ~]# 
2022-04-26 02:54:02 - test_network_sriov_stress.py - [line:348] - root - INFO - Successfully assigned ip to ethernet: ens2f0
2022-04-26 02:54:02 - test_network_sriov_stress.py - [line:621] - root - INFO - start to assign ip for ens2f1 of SUT2: 192.168.10.11
2022-04-26 02:54:08 - sshInstance.py - [line:99] - root - INFO - ifconfig ens2f1 192.168.10.11
[root@localhost ~]# 
2022-04-26 02:54:11 - test_network_sriov_stress.py - [line:348] - root - INFO - Successfully assigned ip to ethernet: ens2f1
2022-04-26 02:54:11 - test_network_sriov_stress.py - [line:145] - root - INFO - virtulize sut1 pf to 4 vfs
2022-04-26 02:54:14 - sshInstance.py - [line:50] - root - INFO - echo 4 > /sys/class/net/ens2f0/device/sriov_numvfs
2022-04-26 02:54:14 - sshInstance.py - [line:54] - root - INFO - 
2022-04-26 02:54:19 - test_network_sriov_stress.py - [line:154] - root - INFO - Successfully Virtualized PF to VFs.
2022-04-26 02:54:22 - sshInstance.py - [line:50] - root - INFO - lspci | grep "Virtual" |grep 4b | awk '{print $1;}'list
2022-04-26 02:54:22 - sshInstance.py - [line:54] - root - INFO - 4b:01.0
4b:01.1
4b:01.2
4b:01.3

2022-04-26 02:54:27 - test_network_sriov_stress.py - [line:168] - root - INFO - Succssfully Gotten vf_id list:

2022-04-26 02:54:30 - sshInstance.py - [line:50] - root - INFO - modprobe vfio-pci

2022-04-26 02:54:30 - sshInstance.py - [line:54] - root - INFO - 
2022-04-26 02:54:35 - test_network_sriov_stress.py - [line:188] - root - INFO - start to bind drive in sut1
2022-04-26 02:54:38 - sshInstance.py - [line:50] - root - INFO - /opt/APP/utility/DPDK_install/dpdk-stable-20.11.1/usertools/dpdk-devbind.py -b vfio-pci 4b:01.0
2022-04-26 02:54:39 - sshInstance.py - [line:54] - root - INFO - 
2022-04-26 02:54:44 - test_network_sriov_stress.py - [line:202] - root - INFO - Successfuly bound VF to vfio-pci
2022-04-26 02:54:47 - sshInstance.py - [line:50] - root - INFO - /opt/APP/utility/DPDK_install/dpdk-stable-20.11.1/usertools/dpdk-devbind.py -b vfio-pci 4b:01.1
2022-04-26 02:54:48 - sshInstance.py - [line:54] - root - INFO - 
2022-04-26 02:54:53 - test_network_sriov_stress.py - [line:202] - root - INFO - Successfuly bound VF to vfio-pci
2022-04-26 02:54:53 - test_network_sriov_stress.py - [line:639] - root - INFO - lspci -Dnn | grep 4b:01.1
2022-04-26 02:54:56 - sshInstance.py - [line:50] - root - INFO - lspci -Dnn | grep 4b:01.1
2022-04-26 02:54:56 - sshInstance.py - [line:54] - root - INFO - 0000:4b:01.1 Ethernet controller [0200]: Intel Corporation Ethernet Adaptive Virtual Function [8086:1889] (rev 02)

2022-04-26 02:55:01 - test_network_sriov_stress.py - [line:651] - root - INFO - Successfully get vf device id: 8086:1889
2022-04-26 02:55:01 - test_network_sriov_stress.py - [line:225] - root - INFO - get default network info: virsh net-info default | grep "Active" | awk '{print $2;}'
2022-04-26 02:55:04 - sshInstance.py - [line:50] - root - INFO - virsh net-info default | grep "Active" | awk '{print $2;}'
2022-04-26 02:55:04 - sshInstance.py - [line:54] - root - INFO - yes

2022-04-26 02:55:09 - test_network_sriov_stress.py - [line:228] - root - INFO - Default network is active? yes

2022-04-26 02:55:09 - test_network_sriov_stress.py - [line:664] - root - INFO - start to create VMs....
2022-04-26 02:55:09 - test_network_sriov_stress.py - [line:258] - root - INFO - try to create VM with the cmd1: virt-install --name VMtest1 --ram 4096 --cpu host --vcpus=5 --network network=default --hostdev pci_0000_4b_01_0 --boot hd --disk path=/home/image/rhel.img,size=50,format=qcow2 --osinfo rhel8.2 --graphics vnc,listen=0.0.0.0,port=5900 --noautoconsole

2022-04-26 02:55:13 - sshInstance.py - [line:75] - root - INFO - virt-install --name VMtest1 --ram 4096 --cpu host --vcpus=5 ---network network=default --hostdev pci_0000_4b_01_0 --boot hd --disk path=/home/iimage/rhel.img,size=50,format=qcow2 --osinfo rhel8.2 --graphics vnc,listen=0.0.0..0,port=5900 --noautoconsole
usage: virt-install --name NAME --memory MB STORAGE INSTALL [options]
virt-install: error: unrecognized arguments: --osinfo rhel8.2
[root@WW25ICX2 ~]# 
2022-04-26 02:55:16 - test_network_sriov_stress.py - [line:261] - root - INFO - cmd1 is not working, try to create VM with the cmd2: virt-install --name VMtest1 --ram 4096 --cpu host --vcpus=5 --network network=default --hostdev pci_0000_4b_01_0 --boot hd --disk path=/home/image/rhel.img,size=50,format=qcow2 --graphics vnc,listen=0.0.0.0,port=5900 --noautoconsole

2022-04-26 02:55:20 - sshInstance.py - [line:75] - root - INFO - virt-install --name VMtest1 --ram 4096 --cpu host --vcpus=5 ---network network=default --hostdev pci_0000_4b_01_0 --boot hd --disk path=/home/iimage/rhel.img,size=50,format=qcow2 --graphics vnc,listen=0.0.0.0,port=5900 --noaautoconsole
2022-04-26 02:55:21 - sshInstance.py - [line:75] - root - INFO - 
** (process:50126): [1;33mWARNING[0m **: [34m02:56:36.707[0m: Entity http://pcisig.com/pci/1B36/0100 referenced but not defined
WARNING  No operating system detected, VM performance may suffer. Specify an OS with --os-variant for optimal results.

Starting install...
2022-04-26 02:55:22 - sshInstance.py - [line:75] - root - INFO - Domain creation completed.
[root@WW25ICX2 ~]# 
2022-04-26 02:55:25 - test_network_sriov_stress.py - [line:275] - root - INFO - Successfully created a VM: VMtest1
2022-04-26 02:55:25 - test_network_sriov_stress.py - [line:258] - root - INFO - try to create VM with the cmd1: virt-install --name VMtest2 --ram 4096 --cpu host --vcpus=5 --network network=default --hostdev pci_0000_4b_01_1 --boot hd --disk path=/home/image/rhel2.img,size=50,format=qcow2 --osinfo rhel8.2 --graphics vnc,listen=0.0.0.0,port=5901 --noautoconsole

2022-04-26 02:55:29 - sshInstance.py - [line:75] - root - INFO - virt-install --name VMtest2 --ram 4096 --cpu host --vcpus=5 ---network network=default --hostdev pci_0000_4b_01_1 --boot hd --disk path=/home/iimage/rhel2.img,size=50,format=qcow2 --osinfo rhel8.2 --graphics vnc,listen=0.0.00.0,port=5901 --noautoconsole
usage: virt-install --name NAME --memory MB STORAGE INSTALL [options]
virt-install: error: unrecognized arguments: --osinfo rhel8.2
[root@WW25ICX2 ~]# 
2022-04-26 02:55:32 - test_network_sriov_stress.py - [line:261] - root - INFO - cmd1 is not working, try to create VM with the cmd2: virt-install --name VMtest2 --ram 4096 --cpu host --vcpus=5 --network network=default --hostdev pci_0000_4b_01_1 --boot hd --disk path=/home/image/rhel2.img,size=50,format=qcow2 --graphics vnc,listen=0.0.0.0,port=5901 --noautoconsole

2022-04-26 02:55:36 - sshInstance.py - [line:75] - root - INFO - virt-install --name VMtest2 --ram 4096 --cpu host --vcpus=5 ---network network=default --hostdev pci_0000_4b_01_1 --boot hd --disk path=/home/iimage/rhel2.img,size=50,format=qcow2 --graphics vnc,listen=0.0.0.0,port=5901 --nooautoconsole

** (process:50191): [1;33mWARNING[0m **: [34m02:56:52.597[0m: Entity http://pcisig.com/pci/1B36/0100 referenced but not defined
2022-04-26 02:55:37 - sshInstance.py - [line:75] - root - INFO - WARNING  No operating system detected, VM performance may suffer. Specify an OS with --os-variant for optimal results.

Starting install...
2022-04-26 02:55:39 - sshInstance.py - [line:75] - root - INFO - Domain creation completed.
[root@WW25ICX2 ~]# 
2022-04-26 02:55:42 - test_network_sriov_stress.py - [line:275] - root - INFO - Successfully created a VM: VMtest2
2022-04-26 02:56:15 - sshInstance.py - [line:50] - root - INFO - virsh domifaddr VMtest1 | grep '192.168' | awk '{print $4;}'
2022-04-26 02:56:15 - sshInstance.py - [line:54] - root - INFO - 192.168.122.28/24

2022-04-26 02:56:21 - test_network_sriov_stress.py - [line:291] - root - INFO - Successfully got VMtest1 IP: 192.168.122.28/24

2022-04-26 02:56:24 - sshInstance.py - [line:50] - root - INFO - virsh domifaddr VMtest2 | grep '192.168' | awk '{print $4;}'
2022-04-26 02:56:24 - sshInstance.py - [line:54] - root - INFO - 192.168.122.115/24

2022-04-26 02:56:29 - test_network_sriov_stress.py - [line:291] - root - INFO - Successfully got VMtest2 IP: 192.168.122.115/24

2022-04-26 02:56:29 - test_network_sriov_stress.py - [line:675] - root - INFO - ip of VM1: 192.168.122.28
2022-04-26 02:56:29 - test_network_sriov_stress.py - [line:676] - root - INFO - ip of VM1: 192.168.122.115
2022-04-26 02:56:29 - test_network_sriov_stress.py - [line:681] - root - INFO - Start ssh VM1 and conf VM1
2022-04-26 02:56:35 - sshInstance.py - [line:99] - root - INFO - ssh-key -R 192.168.122.28
bash: ssh-key: command not found...
[root@WW25ICX2 ~]# 
2022-04-26 02:56:42 - sshInstance.py - [line:75] - root - INFO - ping -c 5 192.168.122.28
PING 192.168.122.28 (192.168.122.28) 56(84) bytes of data.
64 bytes from 192.168.122.28: icmp_seq=1 ttl=64 time=0.314 ms
2022-04-26 02:56:43 - sshInstance.py - [line:75] - root - INFO - 64 bytes from 192.168.122.28: icmp_seq=2 ttl=64 time=0.296 ms
2022-04-26 02:56:44 - sshInstance.py - [line:75] - root - INFO - 64 bytes from 192.168.122.28: icmp_seq=3 ttl=64 time=0.306 ms
2022-04-26 02:56:45 - sshInstance.py - [line:75] - root - INFO - 64 bytes from 192.168.122.28: icmp_seq=4 ttl=64 time=0.298 ms
2022-04-26 02:56:46 - sshInstance.py - [line:75] - root - INFO - 64 bytes from 192.168.122.28: icmp_seq=5 ttl=64 time=0.237 ms

--- 192.168.122.28 ping statistics ---
5 packets transmitted, 5 received, 0% packet loss, time 4136ms
rtt min/avg/max/mdev = 0.237/0.290/0.314/0.029 ms
[root@WW25ICX2 ~]# 
2022-04-26 02:56:55 - sshInstance.py - [line:99] - root - INFO - ssh root@192.168.122.28
root@192.168.122.28's password: 
2022-04-26 02:57:04 - sshInstance.py - [line:99] - root - INFO - 
Activate the web console with: systemctl enable --now cockpit.socket
This system is not registered to Red Hat Insights. See https://cloud.redhat.com/
To register this system, run: insights-client --register
Last login: Mon Apr 25 03:42:05 2022 from 192.168.122.1
[root@localhost ~]# 
2022-04-26 02:57:07 - test_network_sriov_stress.py - [line:375] - root - INFO - Successfully ssh VM.
2022-04-26 02:57:13 - sshInstance.py - [line:99] - root - INFO - lspci -Dnn | grep "8086:1889" --color=never| awk '{print $1;}}'
0000:00:05.0
[root@localhost ~]# 
2022-04-26 02:57:22 - test_network_sriov_stress.py - [line:397] - root - INFO - successfully got vf pci id of vm: 0000:00:05.0
2022-04-26 02:57:28 - sshInstance.py - [line:99] - root - INFO - ls /sys/bus/pci/devices/0000\:00\:05.0/net/ --color=never
ens5
[root@localhost ~]# 
2022-04-26 02:57:33 - test_network_sriov_stress.py - [line:411] - root - INFO - Successfully got vf device name of vm: ens5
2022-04-26 02:57:33 - test_network_sriov_stress.py - [line:423] - root - INFO - try to get ipv6 for ens5
2022-04-26 02:57:33 - test_network_sriov_stress.py - [line:425] - root - INFO - try to get ipv6 in redhat
2022-04-26 02:57:33 - sshInstance.py - [line:187] - root - INFO - starting to delete network of ens5:
2022-04-26 02:57:39 - sshInstance.py - [line:99] - root - INFO - nmcli connection delete ens5
Connection 'ens5' (7415328f-057c-4c4d-8f9e-1d46de42cca2) successfully deleted.
[root@localhost ~]# 
2022-04-26 02:57:42 - sshInstance.py - [line:189] - root - INFO - restart networkManager to make setting valid
2022-04-26 02:57:48 - sshInstance.py - [line:99] - root - INFO - systemctl restart NetworkManager
[root@localhost ~]# 
2022-04-26 02:57:51 - sshInstance.py - [line:192] - root - INFO - starting to add network of ens5:
2022-04-26 02:57:57 - sshInstance.py - [line:99] - root - INFO - nmcli connection add type ethernet con-name ens5 ifname ens5 [A        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [K            5
Connection 'ens5' (f9ca2e16-57db-4f47-8fcf-81ab4f10e72e) successfully added.
[root@localhost ~]# 
2022-04-26 02:58:00 - sshInstance.py - [line:195] - root - INFO - restart networkManager to make setting valid
2022-04-26 02:58:06 - sshInstance.py - [line:99] - root - INFO - systemctl restart NetworkManager
[root@localhost ~]# 
2022-04-26 02:58:15 - sshInstance.py - [line:99] - root - INFO - ifconfig ens5
ens5: flags=4163<UP,BROADCAST,RUNNING,MULTICAST>  mtu 1500
        inet6 fe80::fb2b:d561:d39f:88b7  prefixlen 64  scopeid 0x20<link>
        ether b6:80:ab:8e:24:11  txqueuelen 1000  (Ethernet)
        RX packets 20  bytes 4396 (4.2 KiB)
        RX errors 0  dropped 0  overruns 0  frame 0
        TX packets 41  bytes 6734 (6.5 KiB)
        TX errors 1  dropped 0 overruns 0  carrier 0  collisions 0
[root@localhost ~]# 
2022-04-26 02:58:18 - test_network_sriov_stress.py - [line:438] - root - INFO - assign ip address for VM...
2022-04-26 02:58:24 - sshInstance.py - [line:99] - root - INFO - ifconfig ens5 192.168.10.12
[root@localhost ~]# 
2022-04-26 02:58:27 - test_network_sriov_stress.py - [line:348] - root - INFO - Successfully assigned ip to ethernet: ens5
2022-04-26 02:58:33 - sshInstance.py - [line:99] - root - INFO - exit
logout
Connection to 192.168.122.28 closed.
[root@WW25ICX2 ~]# 
2022-04-26 02:58:36 - test_network_sriov_stress.py - [line:687] - root - INFO - Start ssh VM2 and conf VM2
2022-04-26 02:58:42 - sshInstance.py - [line:99] - root - INFO - ssh-key -R 192.168.122.115
bash: ssh-key: command not found...
[root@WW25ICX2 ~]# 
2022-04-26 02:58:49 - sshInstance.py - [line:75] - root - INFO - ping -c 5 192.168.122.115
PING 192.168.122.115 (192.168.122.115) 56(84) bytes of data.
64 bytes from 192.168.122.115: icmp_seq=1 ttl=64 time=0.241 ms
2022-04-26 02:58:50 - sshInstance.py - [line:75] - root - INFO - 64 bytes from 192.168.122.115: icmp_seq=2 ttl=64 time=0.265 ms
2022-04-26 02:58:51 - sshInstance.py - [line:75] - root - INFO - 64 bytes from 192.168.122.115: icmp_seq=3 ttl=64 time=0.219 ms
2022-04-26 02:58:52 - sshInstance.py - [line:75] - root - INFO - 64 bytes from 192.168.122.115: icmp_seq=4 ttl=64 time=0.255 ms
2022-04-26 02:58:53 - sshInstance.py - [line:75] - root - INFO - 64 bytes from 192.168.122.115: icmp_seq=5 ttl=64 time=0.244 ms

--- 192.168.122.115 ping statistics ---
5 packets transmitted, 5 received, 0% packet loss, time 4090ms
rtt min/avg/max/mdev = 0.219/0.244/0.265/0.025 ms
[root@WW25ICX2 ~]# 
2022-04-26 02:59:02 - sshInstance.py - [line:99] - root - INFO - ssh root@192.168.122.115
root@192.168.122.115's password: 
2022-04-26 02:59:11 - sshInstance.py - [line:99] - root - INFO - 
Activate the web console with: systemctl enable --now cockpit.socket
This system is not registered to Red Hat Insights. See https://cloud.redhat.com/
To register this system, run: insights-client --register
Last login: Mon Apr 25 04:07:20 2022 from 192.168.122.1
[root@f223-102-04-v11 ~]# 
2022-04-26 02:59:14 - test_network_sriov_stress.py - [line:375] - root - INFO - Successfully ssh VM.
2022-04-26 02:59:20 - sshInstance.py - [line:99] - root - INFO - lspci -Dnn | grep "8086:1889" --color=never| awk '{prinnt $1;}'
0000:00:05.0
[root@f223-102-04-v11 ~]# 
2022-04-26 02:59:29 - test_network_sriov_stress.py - [line:397] - root - INFO - successfully got vf pci id of vm: 0000:00:05.0
2022-04-26 02:59:35 - sshInstance.py - [line:99] - root - INFO - ls /sys/bus/pci/devices/0000\:00\:05.0/net/ --color=nevver
ens5
[root@f223-102-04-v11 ~]# 
2022-04-26 02:59:40 - test_network_sriov_stress.py - [line:411] - root - INFO - Successfully got vf device name of vm: ens5
2022-04-26 02:59:40 - test_network_sriov_stress.py - [line:423] - root - INFO - try to get ipv6 for ens5
2022-04-26 02:59:40 - test_network_sriov_stress.py - [line:425] - root - INFO - try to get ipv6 in redhat
2022-04-26 02:59:40 - sshInstance.py - [line:187] - root - INFO - starting to delete network of ens5:
2022-04-26 02:59:46 - sshInstance.py - [line:99] - root - INFO - nmcli connection delete ens5
Connection 'ens5' (64cce50c-e731-4447-8251-a3f5e4132caa) successfully deleted.
[root@f223-102-04-v11 ~]# 
2022-04-26 02:59:49 - sshInstance.py - [line:189] - root - INFO - restart networkManager to make setting valid
2022-04-26 02:59:55 - sshInstance.py - [line:99] - root - INFO - systemctl restart NetworkManager
[root@f223-102-04-v11 ~]# 
2022-04-26 02:59:58 - sshInstance.py - [line:192] - root - INFO - starting to add network of ens5:
2022-04-26 03:00:04 - sshInstance.py - [line:99] - root - INFO - nmcli connection add type ethernet con-name ens5 ifnamee ens5
Connection 'ens5' (122c6c41-16a7-44ee-999a-2acb68217055) successfully added.
[root@f223-102-04-v11 ~]# 
2022-04-26 03:00:07 - sshInstance.py - [line:195] - root - INFO - restart networkManager to make setting valid
2022-04-26 03:00:13 - sshInstance.py - [line:99] - root - INFO - systemctl restart NetworkManager
[root@f223-102-04-v11 ~]# 
2022-04-26 03:00:22 - sshInstance.py - [line:99] - root - INFO - ifconfig ens5
ens5: flags=4163<UP,BROADCAST,RUNNING,MULTICAST>  mtu 1500
        inet6 fe80::9ecb:d0c7:657a:2cfb  prefixlen 64  scopeid 0x20<link>
        ether 8a:53:cd:e7:68:3a  txqueuelen 1000  (Ethernet)
        RX packets 29  bytes 8019 (7.8 KiB)
        RX errors 0  dropped 0  overruns 0  frame 0
        TX packets 62  bytes 10066 (9.8 KiB)
        TX errors 1  dropped 0 overruns 0  carrier 0  collisions 0
[root@f223-102-04-v11 ~]# 
2022-04-26 03:00:25 - test_network_sriov_stress.py - [line:438] - root - INFO - assign ip address for VM...
2022-04-26 03:00:31 - sshInstance.py - [line:99] - root - INFO - ifconfig ens5 192.168.10.13
[root@f223-102-04-v11 ~]# 
2022-04-26 03:00:34 - test_network_sriov_stress.py - [line:348] - root - INFO - Successfully assigned ip to ethernet: ens5
2022-04-26 03:00:40 - sshInstance.py - [line:99] - root - INFO - exit
logout
Connection to 192.168.122.115 closed.
[root@WW25ICX2 ~]# 
2022-04-26 03:00:43 - test_network_sriov_stress.py - [line:695] - root - INFO - Ping two vms IP from SUT:
2022-04-26 03:00:47 - sshInstance.py - [line:75] - root - INFO - ping -c 5 192.168.10.12
PING 192.168.10.12 (192.168.10.12) 56(84) bytes of data.
64 bytes from 192.168.10.12: icmp_seq=1 ttl=64 time=0.224 ms
2022-04-26 03:00:48 - sshInstance.py - [line:75] - root - INFO - 64 bytes from 192.168.10.12: icmp_seq=2 ttl=64 time=0.127 ms
2022-04-26 03:00:49 - sshInstance.py - [line:75] - root - INFO - 64 bytes from 192.168.10.12: icmp_seq=3 ttl=64 time=0.125 ms
2022-04-26 03:00:50 - sshInstance.py - [line:75] - root - INFO - 64 bytes from 192.168.10.12: icmp_seq=4 ttl=64 time=0.143 ms
2022-04-26 03:00:51 - sshInstance.py - [line:75] - root - INFO - 64 bytes from 192.168.10.12: icmp_seq=5 ttl=64 time=0.127 ms

--- 192.168.10.12 ping statistics ---
5 packets transmitted, 5 received, 0% packet loss, time 4095ms
rtt min/avg/max/mdev = 0.125/0.149/0.224/0.038 ms
[root@WW25ICX2 ~]# 
2022-04-26 03:00:58 - sshInstance.py - [line:75] - root - INFO - ping -c 5 192.168.10.13
PING 192.168.10.13 (192.168.10.13) 56(84) bytes of data.
64 bytes from 192.168.10.13: icmp_seq=1 ttl=64 time=0.169 ms
2022-04-26 03:00:59 - sshInstance.py - [line:75] - root - INFO - 64 bytes from 192.168.10.13: icmp_seq=2 ttl=64 time=0.102 ms
2022-04-26 03:01:00 - sshInstance.py - [line:75] - root - INFO - 64 bytes from 192.168.10.13: icmp_seq=3 ttl=64 time=0.102 ms
2022-04-26 03:01:01 - sshInstance.py - [line:75] - root - INFO - 64 bytes from 192.168.10.13: icmp_seq=4 ttl=64 time=0.101 ms
2022-04-26 03:01:02 - sshInstance.py - [line:75] - root - INFO - 64 bytes from 192.168.10.13: icmp_seq=5 ttl=64 time=0.105 ms

--- 192.168.10.13 ping statistics ---
5 packets transmitted, 5 received, 0% packet loss, time 4089ms
rtt min/avg/max/mdev = 0.101/0.115/0.169/0.029 ms
[root@WW25ICX2 ~]# 
2022-04-26 03:01:05 - test_network_sriov_stress.py - [line:703] - root - INFO - ping VMtest2 IP from VMtest1
2022-04-26 03:01:11 - sshInstance.py - [line:99] - root - INFO - ssh-key -R 192.168.122.28
bash: ssh-key: command not found...
[root@WW25ICX2 ~]# 
2022-04-26 03:01:18 - sshInstance.py - [line:75] - root - INFO - ping -c 5 192.168.122.28
PING 192.168.122.28 (192.168.122.28) 56(84) bytes of data.
64 bytes from 192.168.122.28: icmp_seq=1 ttl=64 time=0.292 ms
2022-04-26 03:01:19 - sshInstance.py - [line:75] - root - INFO - 64 bytes from 192.168.122.28: icmp_seq=2 ttl=64 time=0.208 ms
2022-04-26 03:01:20 - sshInstance.py - [line:75] - root - INFO - 64 bytes from 192.168.122.28: icmp_seq=3 ttl=64 time=0.123 ms
2022-04-26 03:01:21 - sshInstance.py - [line:75] - root - INFO - 64 bytes from 192.168.122.28: icmp_seq=4 ttl=64 time=0.217 ms
2022-04-26 03:01:22 - sshInstance.py - [line:75] - root - INFO - 64 bytes from 192.168.122.28: icmp_seq=5 ttl=64 time=0.197 ms

--- 192.168.122.28 ping statistics ---
5 packets transmitted, 5 received, 0% packet loss, time 4097ms
rtt min/avg/max/mdev = 0.123/0.207/0.292/0.055 ms
[root@WW25ICX2 ~]# 
2022-04-26 03:01:31 - sshInstance.py - [line:99] - root - INFO - ssh root@192.168.122.28
root@192.168.122.28's password: 
2022-04-26 03:01:40 - sshInstance.py - [line:99] - root - INFO - 
Activate the web console with: systemctl enable --now cockpit.socket
This system is not registered to Red Hat Insights. See https://cloud.redhat.com/
To register this system, run: insights-client --register
Last login: Mon Apr 25 14:58:15 2022 from 192.168.122.1
[root@localhost ~]# 
2022-04-26 03:01:43 - test_network_sriov_stress.py - [line:375] - root - INFO - Successfully ssh VM.
2022-04-26 03:01:47 - sshInstance.py - [line:75] - root - INFO - ping -c 5 192.168.10.13
PING 192.168.10.13 (192.168.10.13) 56(84) bytes of data.
64 bytes from 192.168.10.13: icmp_seq=1 ttl=64 time=0.255 ms
2022-04-26 03:01:48 - sshInstance.py - [line:75] - root - INFO - 64 bytes from 192.168.10.13: icmp_seq=2 ttl=64 time=0.070 ms
2022-04-26 03:01:49 - sshInstance.py - [line:75] - root - INFO - 64 bytes from 192.168.10.13: icmp_seq=3 ttl=64 time=0.065 ms
2022-04-26 03:01:50 - sshInstance.py - [line:75] - root - INFO - 64 bytes from 192.168.10.13: icmp_seq=4 ttl=64 time=0.059 ms
2022-04-26 03:01:51 - sshInstance.py - [line:75] - root - INFO - 64 bytes from 192.168.10.13: icmp_seq=5 ttl=64 time=0.064 ms

--- 192.168.10.13 ping statistics ---
5 packets transmitted, 5 received, 0% packet loss, time 128ms
rtt min/avg/max/mdev = 0.059/0.102/0.255/0.077 ms
[root@localhost ~]# 
2022-04-26 03:02:00 - sshInstance.py - [line:99] - root - INFO - exit
logout
Connection to 192.168.122.28 closed.
[root@WW25ICX2 ~]# 
2022-04-26 03:02:03 - test_network_sriov_stress.py - [line:711] - root - INFO - ping VMtest2 IP from VMtest1
2022-04-26 03:02:07 - sshInstance.py - [line:75] - root - INFO - ping -c 5 -I 192.168.10.10 192.168.10.11
PING 192.168.10.11 (192.168.10.11) from 192.168.10.10 : 56(84) bytes of data.
64 bytes from 192.168.10.11: icmp_seq=1 ttl=64 time=0.350 ms
2022-04-26 03:02:08 - sshInstance.py - [line:75] - root - INFO - 64 bytes from 192.168.10.11: icmp_seq=2 ttl=64 time=0.160 ms
2022-04-26 03:02:09 - sshInstance.py - [line:75] - root - INFO - 64 bytes from 192.168.10.11: icmp_seq=3 ttl=64 time=0.162 ms
2022-04-26 03:02:10 - sshInstance.py - [line:75] - root - INFO - 64 bytes from 192.168.10.11: icmp_seq=4 ttl=64 time=0.173 ms
2022-04-26 03:02:11 - sshInstance.py - [line:75] - root - INFO - 64 bytes from 192.168.10.11: icmp_seq=5 ttl=64 time=0.157 ms

--- 192.168.10.11 ping statistics ---
5 packets transmitted, 5 received, 0% packet loss, time 4125ms
rtt min/avg/max/mdev = 0.157/0.200/0.350/0.076 ms
[root@WW25ICX2 ~]# 
2022-04-26 03:02:17 - sshInstance.py - [line:50] - root - INFO - echo 0 > /sys/class/net/ens2f0/device/sriov_numvfs
2022-04-26 03:02:19 - sshInstance.py - [line:54] - root - INFO - 
2022-04-26 03:02:24 - test_network_sriov_stress.py - [line:310] - root - INFO - Successfully removed VFs from PF.
2022-04-26 03:02:27 - sshInstance.py - [line:50] - root - INFO - virsh destroy VMtest1
2022-04-26 03:02:27 - sshInstance.py - [line:54] - root - INFO - Domain VMtest1 destroyed


2022-04-26 03:02:32 - test_network_sriov_stress.py - [line:323] - root - INFO - Successfully shut down the VM: VMtest1
2022-04-26 03:02:35 - sshInstance.py - [line:50] - root - INFO - virsh undefine VMtest1
2022-04-26 03:02:35 - sshInstance.py - [line:54] - root - INFO - Domain VMtest1 has been undefined


2022-04-26 03:02:40 - test_network_sriov_stress.py - [line:333] - root - INFO - Successfully removed the VM: VMtest1
2022-04-26 03:02:43 - sshInstance.py - [line:50] - root - INFO - virsh destroy VMtest2
2022-04-26 03:02:44 - sshInstance.py - [line:54] - root - INFO - Domain VMtest2 destroyed


2022-04-26 03:02:49 - test_network_sriov_stress.py - [line:323] - root - INFO - Successfully shut down the VM: VMtest2
2022-04-26 03:02:52 - sshInstance.py - [line:50] - root - INFO - virsh undefine VMtest2
2022-04-26 03:02:52 - sshInstance.py - [line:54] - root - INFO - Domain VMtest2 has been undefined


2022-04-26 03:02:57 - test_network_sriov_stress.py - [line:333] - root - INFO - Successfully removed the VM: VMtest2
.2022-04-26 03:02:57 - test_network_sriov_stress.py - [line:100] - root - INFO - removing VMs: VMtest1, VMtest2
2022-04-26 03:03:00 - sshInstance.py - [line:50] - root - INFO - virsh destroy VMtest2
2022-04-26 03:03:00 - sshInstance.py - [line:54] - root - INFO - error: failed to get domain 'VMtest2'


2022-04-26 03:03:05 - test_network_sriov_stress.py - [line:323] - root - INFO - Successfully shut down the VM: VMtest2
2022-04-26 03:03:08 - sshInstance.py - [line:50] - root - INFO - virsh undefine VMtest2
2022-04-26 03:03:08 - sshInstance.py - [line:54] - root - INFO - error: failed to get domain 'VMtest2'


2022-04-26 03:03:13 - test_network_sriov_stress.py - [line:336] - root - ERROR - Failed to removed the VM VMtest2
2022-04-26 03:03:16 - sshInstance.py - [line:50] - root - INFO - virsh destroy VMtest1
2022-04-26 03:03:16 - sshInstance.py - [line:54] - root - INFO - error: failed to get domain 'VMtest1'


2022-04-26 03:03:21 - test_network_sriov_stress.py - [line:323] - root - INFO - Successfully shut down the VM: VMtest1
2022-04-26 03:03:24 - sshInstance.py - [line:50] - root - INFO - virsh undefine VMtest1
2022-04-26 03:03:24 - sshInstance.py - [line:54] - root - INFO - error: failed to get domain 'VMtest1'


2022-04-26 03:03:29 - test_network_sriov_stress.py - [line:336] - root - ERROR - Failed to removed the VM VMtest1
2022-04-26 03:03:32 - sshInstance.py - [line:50] - root - INFO - echo 0 > /sys/class/net/ens2f0/device/sriov_numvfs
2022-04-26 03:03:32 - sshInstance.py - [line:54] - root - INFO - 
2022-04-26 03:03:40 - sshInstance.py - [line:50] - root - INFO - ifconfig ens2f0 0
2022-04-26 03:03:40 - sshInstance.py - [line:54] - root - INFO - 
2022-04-26 03:03:48 - sshInstance.py - [line:50] - root - INFO - ifconfig ens2f1 0
2022-04-26 03:03:48 - sshInstance.py - [line:54] - root - INFO - 
2022-04-26 03:03:56 - sshInstance.py - [line:50] - root - INFO - dmesg > /tmp/dmesg.log
2022-04-26 03:03:56 - sshInstance.py - [line:54] - root - INFO - 
2022-04-26 03:04:01 - test_network_sriov_stress.py - [line:109] - root - INFO - generate dmesg.log file on sut /tmp/dmesg.log
2022-04-26 03:04:01 - test_network_sriov_stress.py - [line:111] - root - INFO - copy dmesg.log from sut1 /tmp dir to controller dir /root/testconfig/testlogs/2022-04-26/test_PI_Networking_SRIOV_StressDisableEnableFor5TimesWhenVFIsAssigned_VM_Guest[rhel.img-rhel2.img]_02_51_02
2022-04-26 03:04:04 - sshInstance.py - [line:50] - root - INFO - rm -f /tmp/dmesg.log
2022-04-26 03:04:05 - sshInstance.py - [line:54] - root - INFO - 
2022-04-26 03:04:10 - test_network_sriov_stress.py - [line:113] - root - INFO - rm dmesg.log on sut1 /tmp dir
2022-04-26 03:04:13 - sshInstance.py - [line:50] - root - INFO - rm -f /tmp/*.txt
2022-04-26 03:04:13 - sshInstance.py - [line:54] - root - INFO - 
2022-04-26 03:04:18 - sshInstance.py - [line:43] - root - INFO - closing ssh client on SUT
2022-04-26 03:04:18 - NicTestFixture.py - [line:35] - root - INFO - ============================ end of run test_PI_Networking_SRIOV_StressDisableEnableFor5TimesWhenVFIsAssigned_VM_Guest[rhel.img-rhel2.img] case ============================
                  [100%]

=============================== warnings summary ===============================
npxtester/Tests/testNics/test_iperf_client_4hour.py:90
  /root/.jenkins/workspace/test-test-nic-sriov-5times/npxtester/Tests/testNics/test_iperf_client_4hour.py:90: PytestUnknownMarkWarning: Unknown pytest.mark.iperf_tcp - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/mark.html
    @pytest.mark.iperf_tcp

npxtester/Tests/testNics/test_iperf_client_4hour.py:143
  /root/.jenkins/workspace/test-test-nic-sriov-5times/npxtester/Tests/testNics/test_iperf_client_4hour.py:143: PytestUnknownMarkWarning: Unknown pytest.mark.iperf_udp - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/mark.html
    @pytest.mark.iperf_udp

npxtester/Tests/testNics/test_mtu.py:69
  /root/.jenkins/workspace/test-test-nic-sriov-5times/npxtester/Tests/testNics/test_mtu.py:69: PytestUnknownMarkWarning: Unknown pytest.mark.nicSimple - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/mark.html
    @pytest.mark.nicSimple

npxtester/Tests/testNics/test_mtu.py:108
  /root/.jenkins/workspace/test-test-nic-sriov-5times/npxtester/Tests/testNics/test_mtu.py:108: PytestUnknownMarkWarning: Unknown pytest.mark.nicSimple - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/mark.html
    @pytest.mark.nicSimple

npxtester/Tests/testNics/test_network_sriov_stress.py:600
  /root/.jenkins/workspace/test-test-nic-sriov-5times/npxtester/Tests/testNics/test_network_sriov_stress.py:600: PytestUnknownMarkWarning: Unknown pytest.mark.nicSriov5times - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/mark.html
    @pytest.mark.nicSriov5times

npxtester/Tests/testNics/test_network_sriov_stress.py:727
  /root/.jenkins/workspace/test-test-nic-sriov-5times/npxtester/Tests/testNics/test_network_sriov_stress.py:727: PytestUnknownMarkWarning: Unknown pytest.mark.nicSriov2hours - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/mark.html
    @pytest.mark.nicSriov2hours

npxtester/Tests/testNics/test_network_sriov_stress.py:967
  /root/.jenkins/workspace/test-test-nic-sriov-5times/npxtester/Tests/testNics/test_network_sriov_stress.py:967: PytestUnknownMarkWarning: Unknown pytest.mark.SriovVfIperf - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/mark.html
    @pytest.mark.SriovVfIperf

npxtester/Tests/testNics/test_network_sriov_stress.py:1106
  /root/.jenkins/workspace/test-test-nic-sriov-5times/npxtester/Tests/testNics/test_network_sriov_stress.py:1106: PytestUnknownMarkWarning: Unknown pytest.mark.nicIperfTcpSUTGuestStress - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/mark.html
    @pytest.mark.nicIperfTcpSUTGuestStress

npxtester/Tests/testNics/test_network_sriov_stress.py:1107
  /root/.jenkins/workspace/test-test-nic-sriov-5times/npxtester/Tests/testNics/test_network_sriov_stress.py:1107: PytestUnknownMarkWarning: Unknown pytest.mark.SriovVfIperf - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/mark.html
    @pytest.mark.SriovVfIperf

npxtester/Tests/testNics/test_nics.py:119
  /root/.jenkins/workspace/test-test-nic-sriov-5times/npxtester/Tests/testNics/test_nics.py:119: PytestUnknownMarkWarning: Unknown pytest.mark.nicSimple - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/mark.html
    @pytest.mark.nicSimple

npxtester/Tests/testNics/test_nics.py:156
  /root/.jenkins/workspace/test-test-nic-sriov-5times/npxtester/Tests/testNics/test_nics.py:156: PytestUnknownMarkWarning: Unknown pytest.mark.nicSimple - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/mark.html
    @pytest.mark.nicSimple

npxtester/Tests/testNics/test_nics.py:157
  /root/.jenkins/workspace/test-test-nic-sriov-5times/npxtester/Tests/testNics/test_nics.py:157: PytestUnknownMarkWarning: Unknown pytest.mark.connect - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/mark.html
    @pytest.mark.connect

npxtester/Tests/testNics/test_nics.py:189
  /root/.jenkins/workspace/test-test-nic-sriov-5times/npxtester/Tests/testNics/test_nics.py:189: PytestUnknownMarkWarning: Unknown pytest.mark.nicSimple - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/mark.html
    @pytest.mark.nicSimple

npxtester/Tests/testNics/test_nics.py:190
  /root/.jenkins/workspace/test-test-nic-sriov-5times/npxtester/Tests/testNics/test_nics.py:190: PytestUnknownMarkWarning: Unknown pytest.mark.static_ip - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/mark.html
    @pytest.mark.static_ip

npxtester/Tests/testNics/test_nics.py:221
  /root/.jenkins/workspace/test-test-nic-sriov-5times/npxtester/Tests/testNics/test_nics.py:221: PytestUnknownMarkWarning: Unknown pytest.mark.nicSimple - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/mark.html
    @pytest.mark.nicSimple

npxtester/Tests/testNics/test_nics.py:222
  /root/.jenkins/workspace/test-test-nic-sriov-5times/npxtester/Tests/testNics/test_nics.py:222: PytestUnknownMarkWarning: Unknown pytest.mark.test - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/mark.html
    @pytest.mark.test

npxtester/Tests/testNics/test_nics.py:308
  /root/.jenkins/workspace/test-test-nic-sriov-5times/npxtester/Tests/testNics/test_nics.py:308: PytestUnknownMarkWarning: Unknown pytest.mark.nicSimple - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/mark.html
    @pytest.mark.nicSimple

npxtester/Tests/testNics/test_nics.py:334
  /root/.jenkins/workspace/test-test-nic-sriov-5times/npxtester/Tests/testNics/test_nics.py:334: PytestUnknownMarkWarning: Unknown pytest.mark.nicSimple - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/mark.html
    @pytest.mark.nicSimple

npxtester/Tests/testNics/test_nics.py:377
  /root/.jenkins/workspace/test-test-nic-sriov-5times/npxtester/Tests/testNics/test_nics.py:377: PytestUnknownMarkWarning: Unknown pytest.mark.nicSimple - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/mark.html
    @pytest.mark.nicSimple

-- Docs: https://docs.pytest.org/en/stable/warnings.html
- generated html file: file:///root/testconfig/testlogs/HtmlReport/nic_sriov5times_case_report.html -
========== 1 passed, 19 deselected, 19 warnings in 812.27s (0:13:32) ===========
[8mha:////4LeiMYyvaTAszhIQ10pdIeDepJ20hGjpoNTSDQ54vTlOAAAApR+LCAAAAAAAAP9tjTEOwjAQBC9BFLSUPOIi0SCEqNJabniBiY1xYt0Z2yGpeBFf4w8EIlGxxWpnm3m+YJkiHDlabA11jlLjMPjeTgsHjt3F84Atn7FhSuwNSjNI1qaeUXI2MKcoYSFgZajxnBzZDGvRqruqvCJbnXKcvoOA0ukbPKAQkzmrmL8wRthce52Yfp6/AhhDyFBu95/avQGMKkS1vgAAAA==[0m[Pipeline] }
[8mha:////4I/tRoD/KV6lJ76TTEJ8vusLD5mbZiDJ1z7tZBIwhHThAAAApR+LCAAAAAAAAP9tjTEOwjAQBC9BFLSUPOKCQKJBqWgtN7zAJMY4se6C7eBUvIiv8QcsIlGxxWpnm3m9YRk81OwNdpp6S6GxOLjR5IWJfX91nLDjCzZMgZ1GqZPkVp9mlBw1zClKWAhYaWocB0smwlp06qEqp8hU5+jzdxRQ2vYOTyhENkfl4xcmD5vb2Aamn+evAKZhiFDut7l2hw9mQDtDvgAAAA==[0m[Pipeline] // stage
[8mha:////4CAP7WQdC2FMpNczqgZSKdPjM+Zu6Lz59I8VEE3/AlUOAAAAph+LCAAAAAAAAP9tjTEOwjAQBM9BKWgpeYQjBB2iorXc8AITG+PEugv2haTiRXyNPxCIRMVWOyut5vmCMic4UPKycdgGzHWQXez91ORAqb1EGmRDZ1kTZopOajdosu44oyZ2MEcUsFCwdFhHygE9w0o15m6qaNBXJ07TtldQBHuDBwg1mdkk/sKYYH3tbSb8ef4KYOwYxI6h2G4+x/INOibpDcEAAAA=[0m[Pipeline] stage
[8mha:////4Db8sOlSRxZnVxb7xJBDSAteoF0PgC9lkQldzKIJFwYiAAAAph+LCAAAAAAAAP9tjTEOwjAUQ3+LOrAycohUwIiYukZZOEFoQkgb/V9+UtqJE3E17kBLJSY8WLYl673eUESGE7ETjcXWY6y96ELvpiQG4vYaaBANXURNGClYoeygyNhqqYqShUVZDisJa4t1oOjRJdjIRj90GTS68px42o4Scm/u8IRMTuSkOX3LyLC99SYS/jh/ATB2CfLDbrb9/Cw+2YMO4MIAAAA=[0m[Pipeline] { (collect logs)
[8mha:////4LN/0LMxu1hyybTBeE6CHwDx82bv/lkvxFk3cdgHTSuWAAAAox+LCAAAAAAAAP9tjTEOwjAQBC9BFLSUPOIikRJR0VppeIFJjLFj3Tm2g1PxIr7GHwhEomKL1c4283zBOgY4ctBoFfWGYmvQu1HPCzOH/uo4o+ULtkyRncJG5YY7dVqw4aRgSVHCSsBGUes4GtIJtsLKu6ycJF2dU5i/g4DSdAM8oBCzOcmQvjAF2N3GLjL9PH8FMPkEZb3/VO3favo5XL4AAAA=[0m[Pipeline] sh
+ sed -i $'s/\r//g' /root/.jenkins/workspace/test-test-nic-sriov-5times/npxtester/log_analyzer/log_analyzer.sh
[8mha:////4FgvIflgycWoJq7MsZycbV9d7HENRG2WSX14oF+Me/GuAAAApB+LCAAAAAAAAP9tjTEOwjAQBC9BFLSUPOIiAR2iSmu54QUmNsaJdWdsh6TiRXyNPxCIRMUWq51t5vmCZYpw5GixNdQ5So3D4Hs7LRw4dhfPA7Z8xoYpsTcozSBZm3pGydnAnKKEhYCVocZzcmQzrEWr7qryimx1ynH6DgJKp2/wgEJM5qxi/sIYYXPtdWL6ef4KYAwZyt32U/vwBq1seBO+AAAA[0m[Pipeline] sh
+ chmod a+x /root/.jenkins/workspace/test-test-nic-sriov-5times/npxtester/log_analyzer/log_analyzer.sh
[8mha:////4P37N80+NRbzyFlga2j1WDon6H/4VmnQwrRysqQpQX/3AAAApB+LCAAAAAAAAP9tjTEOwjAQBC9BFLSUPOIigagQVVrLDS8wsTFOrDtjOyQVL+Jr/IFAJCq2WO1sM88XLFOEI0eLraHOUWocBt/baeHAsbt4HrDlMzZMib1BaQbJ2tQzSs4G5hQlLASsDDWekyObYS1adVeVV2SrU47TdxBQOn2DBxRiMmcV8xfGCJtrrxPTz/NXAGPIUO62n9qHN+xdYwq+AAAA[0m[Pipeline] sh
+ sh /root/.jenkins/workspace/test-test-nic-sriov-5times/npxtester/log_analyzer/log_analyzer.sh
[8mha:////4P3ih/vRXCzaXCkS1CQDBQXrb3I4EZcCr0NGGH+tKlR4AAAApR+LCAAAAAAAAP9tjTEOwjAQBC9BFLSUPOIigUSDqNJabniBiY1xYt0Z2yGpeBFf4w8EIlGxxWpnm3m+YJkiHDlabA11jlLjMPjeTgsHjt3F84Atn7FhSuwNSjNI1qaeUXI2MKcoYSFgZajxnBzZDGvRqruqvCJbnXKcvoOA0ukbPKAQkzmrmL8wRthce52Yfp6/AhhDyFDu9p/avgEjVhSLvgAAAA==[0m[Pipeline] }
[8mha:////4ISV0Ptmba0km8a80Gq+UClogdZnPDxCJXroOoeI5sULAAAApR+LCAAAAAAAAP9tjTEOwjAQBC9BFLSUPOIiREGBqNJabniBiY1xYt0Z2yGpeBFf4w8EIlGxxWpnm3m+YJkiHDlabA11jlLjMPjeTgsHjt3F84Atn7FhSuwNSjNI1qaeUXI2MKcoYSFgZajxnBzZDGvRqruqvCJbnXKcvoOA0ukbPKAQkzmrmL8wRthce52Yfp6/AhhDyFDu9p/avgE81EHZvgAAAA==[0m[Pipeline] // stage
[8mha:////4LmA5LOJ1yGOO5YQ6RYn+/szIsYEkfxAArsUKSFs0Xa0AAAApB+LCAAAAAAAAP9tjTEOwjAQBDdBFLSUPMIpoEBCVLRWGl5gEmOcWHfBvpBUvIiv8QciIlGx1c4083pjmSKOHJ1qLLWeUuVVF3o3PTVwbK+BB9XwRVVMiYNVpR1Kru1pxpLFYl6WY6GxslQFTp6cYK0b8zBFMOSKs8TJHTRyX9/xRKanspgoXxgjNre+Tky/zt8Axq4T5Nu9INt9AE/B/a+9AAAA[0m[Pipeline] }
[8mha:////4KVKO0UZMawo8C9tjXN4L1+ryWbU4zmvD2sDlsKoyYtXAAAAox+LCAAAAAAAAP9tjTESgjAQRT84FraWHiIUVI5jZZuh8QQRYgxkdjFZhMoTeTXvICMzVv7qv9e81xvrFHHk6FRrqfOUaq/6MLj5qZFjdw08qpYvqmZKHKyq7FhxY08LViwWy7IcK42NpTpw8uQEW92ahymCIVecJc7uoJH75o4nMj2XxUT5whSxuw1NYvp1/gYw9b0gL/eCrPwAXH35DL0AAAA=[0m[Pipeline] // node
[8mha:////4A8vvhfR6oegt+2XAbvn3Z0Fx6j3lb1QUh2abilCs+OGAAAAox+LCAAAAAAAAP9tjTEOwjAQBDdBFLSUPMJBiA5R0VppeIFJjHFi3QX7QlLxIr7GH4iIRMVWO9PM641lijhydKqx1HpKlVdd6N301MCxvQYeVMMXVTElDlaVdii5tqcZSxaLeVmOhcbKUhU4eXKCtW7MwxTBkCvOEid30Mh9fccTmZ7KYqJ8YYzY3Po6Mf06fwMYu06Q77eCbPcBAx/ra70AAAA=[0m[Pipeline] End of Pipeline
Finished: SUCCESS
